wordle-web/pages/game.vue

```diff
@@ -0,0 +1,62 @@
+<template>
+  <v-card class="mx-auto my-8" elevation="16" max-width="500">
+    <v-card-item>
+      <v-card-title> Would you want to light up your day? </v-card-title>
+      <v-card-subtitle> (y/n) </v-card-subtitle>
+    </v-card-item>
+
+    <v-text-field
+      v-if="!jokeAsked"
+      v-model="input"
+      label="Obviously Yes:)"
+      outlined
+      clearable />
+    <v-text-field
+      v-else
+      v-model="punchline"
+      label="Why don't scientists trust atoms?"
+      outlined
+      clearable />
+
+    <v-card-actions>
+      <v-spacer />
+      <v-btn
+          color="blue"
+          variant="elevated"
+          elevation="8"
+          @click="jokeAsked ? checkPunchline() : askForJoke()">
+          Enter
+      </v-btn>
```

Comment:
It's cool you made your own game for this assignment. My partner and I just kept the Wordle code since that's the focus of assignment 2, but making your own game is a fun choice!

Reply:
Thank you!